src/Section1/Section1.js

```diff
@@ -1,15 +1,14 @@
-import { Container, Title, Image } from './style';
-
-
+import { Container, Title, Image, HiddenOnMobile } from "./style";
 
 export const Section1 = () => {
-    return(
-        <Container>
-            <Title>Learning how to code</Title>
-            <Image />
-            
-        </Container>
-    )
-}
-
-export default Section1;
\ No newline at end of file
+  return (
+    <Container>
+      <Title>
+        Learning <HiddenOnMobile>how</HiddenOnMobile> to code
+      </Title>
+      <Image />
```

Comment:
[Accessibility] Since this illustration is just decorative you can add an attribute named "aria-hidden" to it, so assertive technologies (like a screen reader for people who can't see your website) won't pick up on it. 

https://www.w3.org/WAI/tutorials/images/decorative/